src/app/components/pages/MyProgress.tsx

```diff
@@ -209,40 +186,7 @@ const MyProgress = withRouter((props: MyProgressProps) => {
                                 <LinkToContentSummaryList items={progress.oldestIncompleteQuestions}/>
                             </Col>}
                         </Row>
-                    </div>,
-                    ...(isPhy && viewingOwnData && isTeacherOrAbove(user) && {"Teacher Activity": <div>
-                        <TeacherAchievement
-                            verb="created"
-                            count={achievements && achievements.TEACHER_GROUPS_CREATED}
-                            item="group"
-                            createMoreText="Manage groups"
-                            createMoreLink="/groups"
-                            iconClassName="group-badge"/>
-
-                        <TeacherAchievement
```

Comment:
You may be already doing this, but just in case: when removing unneeded code it's also worth checking whether all the things you've removed are still needed. For instance, the `achievements` variable can be removed now. And the same about `TeacherAchievement`, as this was the only place where it was used.

Reply:
I spotted `achievements` in the siteSpecific ticket so that is gone. I hadn't spotted TeacherAchievement but I've deleted that page now in the next ticket! As much as possible I'm trying to remove variables / do simple refactoring as I go along

Reply:
In that case, removing it as part of this ticket should not cause any conflicts once you rebase the other ticket. But in any case, this is the right PR to get rid of those. It's a PITA, I know, but this is one of the reasons why it's better not to work on a feature that depends on a branch that hasn't been merged yet.

Reply:
No problem I'll pop it in this branch also